src/main/java/frc/robot/hardware/phoenix6/Phoenix6DeviceID.java

```diff
@@ -0,0 +1,9 @@
+package frc.robot.hardware.phoenix6;
+
+public record Phoenix6DeviceID(int ID, BusChain busChain) {
+
+	public Phoenix6DeviceID(int ID) {
+		this(ID, BusChain.ROBORIO);
```

Comment:
what if canivore?

Reply:
its a record that can get BusChain  that can be CANIVORE. its just a default constructor